game-backend/src/IntegrationTest.test.ts

```diff
@@ -23,7 +23,7 @@ const performTickNTimes = (
   userContextContainers: UserContextContainer,
   times: number
 ): State => {
-  let state = startState;
+  let state: State = deepClone(startState);
```

Comment:
`deepClone` is not necessary - the engine is pure and does not modify the parameters in any way.
That's what @sir-marc wanted to say :)